pkg/apis/crd/v1alpha2/types.go

```diff
@@ -221,3 +229,46 @@ type EgressList struct {
 
 	Items []Egress `json:"items"`
 }
+
+// +genclient
+// +genclient:nonNamespaced
+// +genclient:noStatus
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// ExternalIPPool defines one or multiple IP sets that can be used in external network. For instance, the IPs can be
```

Comment:
"in *the* external network"?
or "in external networks"?

Reply:
done